### PR TITLE
Comments, JavaScript and Button Styling

### PR DESCRIPTION
**Pull request by mjb44552**
- Added comments to login page to make code more readable
- Moved Java Script code for generating random job-seeking quotes into the layout.html document. 
- Changed some classes of some buttons to make their styling more consistent with what is already on the page. 

Note: to maintain a separation of concerns would we consider creating a scripts folder which contains the files for all the Java Script which our platform will use? Currently our scripts are just spread around on the site and  thus making it difficult to understand how they integrate into the existing code.

### CODE REVIEW

File: app/templates/layout.html [hunk 1]
```diff
@@ -99,5 +100,40 @@
         setInterval(refreshContent, 5000); // 5000 milliseconds = 5 seconds
       });
     </script>
+
+    <!-- script for login page quotes-->
```
Comment (saundlj) [inline app/templates/layout.html]: I don't think this script should be added to the layout.html file... would it not be better off putting into the page that's actually going to use it? lets try keep the layout page to consistent elements for ALL pages.

**Review by saundlj — CHANGES_REQUESTED** [review]
Styling changes look decent. I suggest moving the java script quote code to the specific page it will be in rather than in the template page.

File: app/templates/layout.html (hunk 1 above)
Comment (mjb44552) [reply app/templates/layout.html]: Done, adjusted on another commit I did to this branch. Can merge this extend branch to main later.

**Comment by vittopan** [comment]
Looks mint mate, hopefully we can merge without any conflicts.

**Comment by tomve1tch** [comment]
Old pull request... unsure why it was never merged.

**Pull request closed without merging**